packages/calypso-build/bin/cb.js

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
+
+if ( ! process.argv.some( arg => arg.startsWith( '--config' ) ) ) {
+	process.argv.push( '--config', require.resolve( '@automattic/calypso-build/webpack.config.js' ) );
```

Comment:
This could be relative to this file instead of resolving. It probably makes sense to ensure we're using the webpack config relative to the bin.

Reply:
60b9d96